scripts/beck/st/error_injection.py

```diff
@@ -1,66 +1,78 @@
+logger.addHandler(ch)
+
+logger.info("***** Script Started *****")
 
 parser = argparse.ArgumentParser(description="Basic Arguments to run the script")
 parser.add_argument('-rn', action='store_true', default=False, dest='random', help='For inducing error at Random place')
 parser.add_argument('-e', action='store', default= 0, type=int, dest='noOfErr', help='How Many number of error do you want to induce in Metadata')
-parser.add_argument('-rt', action='store', dest='recordT', help='Record Type For inducing error at perticular record like BE_BTREE, BE_EMAP, CAS_CTG etc')
-parser.add_argument('-m', action='store', dest='file', help='Metadata Path')
+parser.add_argument('-rt', action='store', dest='Record_Type', help='Record Type For inducing error at perticular record like BE_BTREE, BE_EMAP, CAS_CTG etc')
+parser.add_argument('-m', action='store', dest='mfile', help='Metadata Path')
 parser.add_argument('-v', action='store_true', default=False, dest='verify', help='Read full Metadata and Print all the Records entry counts')
 parser.add_argument('-a', action='store_true', default=False, dest='allErr', help='Induce Error in All Record at Random place')
 parser.add_argument('-gmd', action='store_true', default=False, dest='allGMD', help='Induce Error in All GMD type of Record at Random place')
 parser.add_argument('-dmd', action='store_true', default=False, dest='allDMD', help='Induce Error in All DMD type of Record at Random place')
 parser.add_argument('-512k', action='store_true', default=False, dest='err512k', help='Induce 512K bytes error in Metadata')
 parser.add_argument('-huge', action='store_true', default=False, dest='hugeCorruption', help='Induce Huge amount of corruption in Metadata')
+parser.add_argument('-seed', action='store', default=0, type=float, dest='seed', help='Seed for generating Random errors corruption')
 
 args = parser.parse_args();
-#print("Random {0} No of Errors {1} Record Types {2} File name {3} Verify {4}".format(args.random, args.noOfCorr, args.recordType, args.filename, args.verify ))
 
 results = parser.parse_args()
-print('Induce Random Error        = {!r}'.format(args.random))
-print('No of Error induce         = {!r}'.format(args.noOfErr))
-print('Record Type                = {!r}'.format(args.recordT))
-print('Metadata file path         = {!r}'.format(args.file))
-print('Verify Record entries      = {!r}'.format(args.verify))
-print('Induce Error in All Record = {!r}'.format(args.allErr))
-print('Induce Error in GMD Record = {!r}'.format(args.allGMD))
-print('Induce Error in DMD Record = {!r}'.format(args.allDMD))
-print('Induce 512k errors         = {!r}'.format(args.err512k))
-print('Induce huge corruption     = {!r}'.format(args.hugeCorruption))
-
-filename  = args.file
-recordType= args.recordT
+logger.info('Induce Random Error        = {!r}'.format(args.random))
+logger.info('No of Error induce         = {!r}'.format(args.noOfErr))
+logger.info('Record Type                = {!r}'.format(args.Record_Type))
```

Comment:
Record_Type?
The name of this field is using different naming rules.
recordType?

Also, "Record_Type" has a different color here in the PR. Why? Is it a keyword? a reserved word? A special or already used word?

Reply:
I don't know why is it so. This is not a keyword, reserved or special word.